src/AutoRest.CSharp/Mgmt/Models/MgmtClientOperation.cs

```diff
@@ -134,14 +134,23 @@ public MethodSignature MethodSignature
         private FormattableString BuildDescription()
         {
             var pathInformation = _operations.Select(operation =>
-                (FormattableString)$@"<item>
+            {
+                string resourceTypeItem = String.Empty;
```

Comment:
well in this repo we prefer the keyword instead of their real type names for cases like this.
```suggestion
                string resourceTypeItem = string.Empty;
```
like `string` vs `System.String`, `int` vs `System.Int32`, `float` vs `System.Single`, etc.
Those real type names require you to include `using System`, but the keywords do not.